projects/core/src/occ/adapters/product/occ-product.adapter.ts

```diff
@@ -26,6 +27,36 @@ export class OccProductAdapter implements ProductAdapter {
     protected requestsOptimizer: OccRequestsOptimizerService
   ) {}
 
+  loadRealTimeStock(productCode: string): Observable<string> {
```

Comment:
MAJOR: to avoid 2 calls to the same api every time we need stock information, it is better to alter the return value to Observable<{ quantity: string; availability: string }> so that we can get both information in single API call

Reply:
done